### PR TITLE
Release v0.9.3 to master

### PR DESCRIPTION
**Pull request by djrtwo**
[DO NOT MERGE]

Detailed notes will be available in release.

### CODE REVIEW

File: specs/core/0_beacon-chain.md [hunk 1]
```diff
@@ -472,6 +482,34 @@ class BeaconState(Container):
     finalized_checkpoint: Checkpoint
 ```
 
+### Signed envelopes
+
+Some messages in the protocol are wrapped in an envelop to better facilitate adding/pruning the signature and to `hash_tree_root` the `message` separate from the signature.
```
Comment (MrChico) [inline specs/core/0_beacon-chain.md]:
```suggestion
Some messages in the protocol are wrapped in an envelope to better facilitate adding/pruning the signature and to `hash_tree_root` the `message` separate from the signature.  
```

Comment (MrChico) [reply specs/core/0_beacon-chain.md]: > The verb envelop (without an e on the end) means to surround or to enclose. It is sometimes confused with the noun envelope, which most commonly refers to a flat rectangular paper container for a letter.

File: specs/core/0_fork-choice.md [hunk 2]
```diff
@@ -136,17 +136,72 @@ def get_latest_attesting_balance(store: Store, root: Root) -> Gwei:
     ))
 ```
 
+#### `filter_block_tree`
```
Comment (MrChico) [inline specs/core/0_fork-choice.md]: missing in TOC

File: specs/core/0_beacon-chain.md (hunk 1 above)
Comment (djrtwo) [reply specs/core/0_beacon-chain.md]: envelop it in an envelope

File: specs/core/0_fork-choice.md (hunk 2 above)
Comment (djrtwo) [reply specs/core/0_fork-choice.md]: Thanks! fixed the fork choice TOC in #1510 which is under review for this release

**Comment by djrtwo** [comment]
removed. Thanks @terencechain !

**Pull request merged**